src/main/kotlin/no/nav/syfo/dialogmote/api/v2/DialogmoteApiV2.kt

```diff
@@ -43,6 +46,16 @@ fun Route.registerDialogmoteApiV2(
                 call.respond(dialogmoteDTOList)
             }
         }
+        get(dialogmoteApiVeilederIdentUrlPath) {
+            val token = getBearerHeader()
+                ?: throw IllegalArgumentException("No Authorization header supplied")
+
```

Comment:
Tror kanskje det bør være med noe ganske likt det med "inkluderHistoriske" i DialogmoteEnhetApiV2. Dette slik at man kan få ut pågående møter, og ikke hele historikken inkludert gamle møter.

Reply:
Ja, vi burde nok ha tilsvarende filtrering i sqlen for å hente veileders møter som vi har når vi henter enhetens møter. (dvs default bare "aktive" møter)

Reply:
Greit å gjøre det slik det er blitt nå også: dvs hente møter med status innkalt/nytt_tid_sted. Kan evnt utvide for å kunne hente historiske møter senere om behovet skulle oppstå.